### PR TITLE
Patch for broken private Github API use and repo cloning

### PR DESCRIPTION
**Pull request by hedss**
Connects to #31 
Connects to #32

### CODE REVIEW

**Comment by hedss** [comment]
Test note: These fixes have been tested on the live VersionBot Integration via the use of both a private and public test repo.

File: lib/bots/versionbot.ts [hunk 1]
```diff
@@ -422,7 +428,7 @@ export class VersionBot extends GithubBot.GithubBot {
         };
 
         return Promise.mapSeries([
-            `git clone https://${process.env.WEBHOOK_SECRET}:x-oauth-basic@github.com/${versionData.repoFullName} ` +
+            `git clone https://${this.authToken}:${this.authToken}@github.com/${versionData.repoFullName} ` +
```
Comment (CameronDiver) [inline lib/bots/versionbot.ts]: I assume you've tested, but should this be repeated here?

File: lib/bots/githubbot.ts [hunk 2]
```diff
@@ -61,7 +62,7 @@ export class GithubBot extends ProcBot.ProcBot<string> {
         // The `github` module is a bit behind the preview API. We may have to override
         // some of the methods here (PR review comments for a start).
         this.githubApi = new GithubApi({
-            // debug: true,
+            //debug: true,
```
Comment (CameronDiver) [inline lib/bots/githubbot.ts]: Nit: Comment should have a space after `//`, perhaps this one shouldn't be there at all though?

File: lib/bots/versionbot.ts (hunk 1 above)
Comment (hedss) [inline lib/bots/versionbot.ts]: Yes, this is the reason, in fact, private repos didn't clone. See #32

**Pull request merged**